README.md

```diff
@@ -7,26 +7,45 @@
 
 Fastify Redis connection plugin; with this you can share the same Redis connection in every part of your server.
 
-Under the hood [ioredis](https://github.com/luin/ioredis) is used as client, the ``options`` that you pass to `register` will be passed to the Redis client.
-
 ## Install
+
 ```
 npm i fastify-redis --save
 ```
+
 ## Usage
+
 Add it to your project with `register` and you are done!
-You can access the *Redis* client via `fastify.redis`. The client is
-automatically closed when the fastify instance is closed.
 
-```js
-'use strict'
+### Create a new Redis Client
 
+Under the hood [ioredis](https://github.com/luin/ioredis) is used as client, the ``options`` that you pass to `register` will be passed to the Redis client.
+
+```js
 const fastify = require('fastify')()
 
+// create by specifying host
 fastify.register(require('fastify-redis'), { host: '127.0.0.1' })
-// or
+
+// OR by specifying Redis URL
 fastify.register(require('fastify-redis'), { url: 'redis://127.0.0.1', /* other redis options */ })
 
+// OR with more options
+fastify.register(require('fastify-redis'), { 
+  host: '127.0.0.1', 
+  password: '***',
+  port: 6379, // Redis port
+  family: 4   // 4 (IPv4) or 6 (IPv6)
+})
+```
+
+### Accessing the Redis Client
+
+Once you have registered your plugin, you can access the Redis client via `fastify.redis`. 
+
+The client is automatically closed when the fastify instance is closed.
+
+```js
```

Comment:
```suggestion
```js
'use strict'

const Fastify = require('fastify')
const fastifyRedis = require('fastify-redis')

const fastify = Fastify({ logger: true })

fastify.register(fastifyRedis, { 
  host: '127.0.0.1', 
  password: 'your strong password here',
  port: 6379, // Redis port
  family: 4   // 4 (IPv4) or 6 (IPv6)
})

```

Missed that this example was cut.

Reply:
That will teach me to blinding click "commit suggestion" 🙈